libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2114,10 +2114,12 @@ SecondarySourceLocation SMTEncoder::callStackMessage(vector<CallStackEntry> cons
 {
 	SecondarySourceLocation callStackLocation;
 	solAssert(!_callStack.empty(), "");
-	callStackLocation.append("Callstack: ", SourceLocation());
+	callStackLocation.append("Callstack:\n", SourceLocation());
 	for (auto const& call: _callStack | boost::adaptors::reversed)
 		if (call.second)
 			callStackLocation.append("", call.second->location());
+	if (callStackLocation.infos.size() == 1)
+		return {};
```

Comment:
Why this change?

I don't see the callstack printed in any of the tests so I can't tell if it's right or wrong...

Reply:
I changed how the secondarysource location is printed, so maybe this won't be necessary